tutorials/notebooks/example_quick_start_torchvision.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "c9e7b10d2bfe67d4",
+      "metadata": {
+        "collapsed": false,
+        "id": "c9e7b10d2bfe67d4"
+      },
+      "source": [
+        "# Quick Start\n",
+        "\n",
+        "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/quickstart-table/tutorials/notebooks/example_quick_start_torchvision.ipynb)\n",
+        "\n",
+        "Steps:\n",
```

Comment:
Expand this section a little bit:

Overview
This tutorial shows how to use "quick-start" with a pre-trained model from the torchvision library.
The following steps will be covered:
...the steps you mentioned...